webpack.config.js

```diff
@@ -414,6 +414,14 @@ module.exports = {
                     }
                 }]
             },
+            {
+                test: /\.(eot|woff|woff2|svg|ttf)([\?]?.*)$/,
+                loader: 'file-loader',
+            },
+            {
+                test: /\.(jpg)$/i,
+                loader: 'url-loader',
```

Comment:
Likewise, do tables use any .jpgs for styling actually? Do the tables here look any different without this extra loader?

Reply:
Without this webpack fails with errors like the following.
```
    ERROR in ./node_modules/@patternfly/react-styles/css/assets/images/pfbg_576.jpg 1:0
    Module parse failed: Unexpected character '�' (1:0)
    You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
    (Source code omitted for this binary file)
     @ ./node_modules/@patternfly/react-styles/css/components/BackgroundImage/background-image.css (./node_modules/css-loader?minimize=&root=/home/kkoukiou/repos/cockpit-bak/pkg/lib!./node_modules/@patternfly/react-styles/css/components/BackgroundImage/background-image.css) 7:274-317
    
    ERROR in ./node_modules/@patternfly/react-styles/css/assets/images/pfbg_576@2x.jpg 1:0
    Module parse failed: Unexpected character '�' (1:0)
    You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
    (Source code omitted for this binary file)
     @ ./node_modules/@patternfly/react-styles/css/components/BackgroundImage/background-image.css (./node_modules/css-loader?minimize=&root=/home/kkoukiou/repos/cockpit-bak/pkg/lib!./node_modules/@patternfly/react-styles/css/components/BackgroundImage/background-image.css) 7:388-434
    
    ERROR in ./node_modules/@patternfly/react-styles/css/assets/images/pfbg_768.jpg 1:0
    Module parse failed: Unexpected character '�' (1:0)
    You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
    (Source code omitted for this binary file)
     @ ./node_modules/@patternfly/react-styles/css/components/BackgroundImage/background-image.css (./node_modules/css-loader?minimize=&root=/home/kkoukiou/repos/cockpit-bak/pkg/lib!./node_modules/@patternfly/react-styles/css/components/BackgroundImage/background-image.css) 7:506-549
    
    ERROR in ./node_modules/@patternfly/react-styles/css/assets/images/pfbg_768@2x.jpg 1:0
    Module parse failed: Unexpected character '�' (1:0)
    You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
    (Source code omitted for this binary file)
     @ ./node_modules/@patternfly/react-styles/css/components/BackgroundImage/background-image.css (./node_modules/css-loader?minimize=&root=/home/kkoukiou/repos/cockpit-bak/pkg/lib!./node_modules/@patternfly/react-styles/css/components/BackgroundImage/background-image.css) 7:624-670
    

```